### PR TITLE
add single task desgin 

### LINKED ISSUE #112: View single task desgin
Add desgin and CSS for single task page

### PR DESCRIPTION
**Pull request by avraham1992**
Fixes #112 (**#115** its close )
## Changes you made

- when it's writing a future option to delete or edit the task
- add single task design (by the design that @aradz1997 do -\+ )

### CODE REVIEW

**Comment by kobihk** [comment]
> * desgin

**Comment by kobihk** [comment]
> * add single task desgin(by the desgin that @aradz1997 do -+ )

@avraham1992 please fix the typo: * design in the PR description

File: tasks/templates/tasks/view_single_task.html [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
+                            priority: {{task.priority}}<b>|</b>
+                            task id :{{ task.id }}
+                            {% if user.is_staff %}
+                            <b>|</b> {{ task.assignee.team }} 
+                            {% endif %}
+                        </small>
+                    </div>
+                    <h3><a class="article_title" href="#">{{ task.title }}</a></h3>
+                    <p class="article-content">{{ task.description }}</p>
+                    {% if  user.is_manager %}
```
Comment (kobihk) [inline tasks/templates/tasks/view_single_task.html:21]: Please remove redundant space
```suggestion
                    {% if user.is_manager %}
```

File: tasks/templates/tasks/view_single_task.html [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
+    {% if user.user.is_authenticated %}
+        <h2>Tasks</h2>
+            <article class="media content-section">
+                <div class="media_body">
+                    <div class="article_metadata">
+                        <small class="text-muted">
+                            Created on: {{ task.created_date|date:"F d, Y" }} <b>|</b>
+                            Created by: {{ task.created_by }} <b>|</b>
+                            Assignee: {{ task.assignee }}<b>|</b>
+                            priority: {{task.priority}}<b>|</b>
```
Comment (kobihk) [inline tasks/templates/tasks/view_single_task.html:12]: Please add spaces to be consistent with the code style in this file
```suggestion
                            priority: {{ task.priority }}<b>|</b>
```

**Review by kobihk — CHANGES_REQUESTED** [review]

File: tasks/templates/tasks/view_single_task.html [hunk 3]
```diff
@@ -18,12 +18,12 @@ <h2>Tasks</h2>
                     </div>
                     <h3><a class="article_title" href="#">{{ task.title }}</a></h3>
                     <p class="article-content">{{ task.description }}</p>
-                    {% if  user.is_manager %}
-					 	          <div>
-					 	          	<a class="btn btn-secondary btn-sm mt-1 md-1" href="#">Update</a>
-						            <a class="btn btn-danger btn-sm mt-1 md-1" href="#">Detele</a>
-					 	          </div>
-				          	{% endif %}
+                     {% if  user.is_manager %}
```
Comment (kobihk) [inline tasks/templates/tasks/view_single_task.html:21]: I meant the redundant space between the "if" to "user.is_manager"
```suggestion
                     {% if user.is_manager %}
```

**Review by kobihk — CHANGES_REQUESTED** [review]

**Review by kobihk — APPROVED** [review]

File: tasks/templates/tasks/view_single_task.html [hunk 4]
```diff
@@ -1,19 +1,29 @@
...
-    <form action="" method="POST">
-      {% csrf_token %} {{form}}
-      <input type="submit" />
-    </form>
-  </div>
-</div>
+    {% if user.user.is_authenticated %}
+        <h2>Tasks</h2>
+            <article class="media content-section">
+                <div class="media_body">
```
Comment (natifridman) [inline tasks/templates/tasks/view_single_task.html:6]: This div isn't closed

File: tasks/templates/tasks/view_single_task.html [hunk 5]
```diff
@@ -1,19 +1,29 @@
...
+                    </div>
+                    <h3><a class="article_title" href="#">{{ task.title }}</a></h3>
+                    <p class="article-content">{{ task.description }}</p>
+                     {% if user.is_manager %}
+		      <div>
+		    	<a class="btn btn-secondary btn-sm mt-1 md-1" href="#">Update</a>
+		        <a class="btn btn-danger btn-sm mt-1 md-1" href="#">Detele</a>
+		      </div>
+		     {% endif %}
+                </article>
```
Comment (natifridman) [inline tasks/templates/tasks/view_single_task.html:27]: Please fix indentation

**Review by natifridman — CHANGES_REQUESTED** [review]

**Review by natifridman — APPROVED** [review]

**Review by kobihk — APPROVED** [review]

**Review by orzionpour — APPROVED** [review]

**Review by aradz1997 — APPROVED** [review]

**Pull request merged**